lib/manageiq/environment.rb

```diff
@@ -19,7 +19,16 @@ def self.manageiq_plugin_setup
         create_database_user
       end
 
-      setup_test_environment
+      setup_test_environment(:task_prefix => 'app:', :root => plugin_root)
+    end
+
+    def self.manageiq_plugin_update
```

Comment:
What does this buy us?  Just the `bundle update`?  If that's the case, I'd rather see `manageiq_plugin_setup` always `bundle update` so that we don't have duplicate logic.